src/components/nanopub/PreviewNanopub.tsx

```
@@ -0,0 +1,166 @@
+
+  if (previewDescription) {
+    return (
+      <div>
+        <Button
+          onClick={() => setPreviewDescription("")}
+          icon={"arrow-left"}
+          text={"Back"}
+          className="mb-4"
+          outlined
+        />
+        <div dangerouslySetInnerHTML={{ __html: previewDescription }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)